scene/resources/font.cpp

```diff
@@ -502,6 +502,11 @@ void FontData::set_data(const PackedByteArray &p_data) {
 }
 
 PackedByteArray FontData::get_data() const {
+	if (unlikely((size_t)data.size() != data_size)) {
```

Comment:
If feels weird that this could happen and be handled in the getter specifically. Shouldn't it be properly matching if `set_data()` was called?

Reply:
It could be handled in the `set_data_ptr`, but then it will create extra copy of all editor fonts every time (more than 10 MB of wasted memory), this way `PackedByteArray` copy is created on demand.